clouddq/integration/bigquery/dq_target_table_utils.py

```diff
@@ -65,7 +78,9 @@ def load_target_table_from_bigquery(
         SELECT * from `{dq_summary_table_name}`
         WHERE invocation_id='{invocation_id}'
         AND DATE(execution_ts)='{partition_date}'"""
-        bigquery_client.execute_query(query_string=query_string)
+        summary_data = bigquery_client.execute_query(query_string=query_string).result()
```

Comment:
are you sure executing a DML CREATE TABLE AS SELECT returns RowIterator?

Reply:
It does but it's an EmptyRowIterator :-/ I have refactored a bit to run first the SELECT and get the data, then CREATE TABLE to create the target summary table.